docs/abstractions.py

```diff
@@ -266,7 +266,9 @@ def linearize(self): pass
   uops: List[UOp]
 
 from tinygrad.tensor import Tensor
-result = Tensor(2) + Tensor(3)
+from tinygrad.helpers import prod
+result = Tensor(2).realize() + Tensor(3).realize()
+result.lazydata.realized = Device[Device.DEFAULT].buffer(prod(result.shape), result.dtype)
```

Comment:
I was confused about this line when reading the file. Maybe add a comment next time. 🙏